src/tasks/run_closed_loop.py

```diff
@@ -16,10 +46,20 @@ def run():
         print('pip install "neural-data-simulator[extras]"')
         return
 
-    encoder = subprocess.Popen(["encoder"])
-    ephys = subprocess.Popen(["ephys_generator"])
-    decoder = subprocess.Popen(["decoder"])
-    center_out_reach = subprocess.Popen(["center_out_reach"])
+    args = _parse_args()
+
+    SETTINGS_PATH_PARAM = "--settings-path"
+
+    nds_params = _build_param_from_arg(args, "nds_settings_path", SETTINGS_PATH_PARAM)
+    decoder_params = _build_param_from_arg(
+        args, "decoder_settings_path", SETTINGS_PATH_PARAM
+    )
+    task_params = _build_param_from_arg(args, "task_settings_path", SETTINGS_PATH_PARAM)
```

Comment:
You don't need the `getattr` inside `_build_param_from_arg`, if the argument is not passed to the script you will have None as value
```suggestion
    nds_params = _build_param_from_arg(args.nds_settings_path, SETTINGS_PATH_PARAM)
    decoder_params = _build_param_from_arg(
        args.decoder_settings_path, SETTINGS_PATH_PARAM
    )
    task_params = _build_param_from_arg(args.task_settings_path, SETTINGS_PATH_PARAM)
```

Reply:
Nice! I've already sent the update.